bedrock-entry.sh

```diff
@@ -111,7 +111,7 @@ if [[ ! -f "bedrock_server-${VERSION}" ]]; then
   TMP_ZIP="$TMP_DIR/$(basename "${DOWNLOAD_URL}")"
 
   echo "Downloading Bedrock server version ${VERSION} ..."
-  if ! curl "${curlArgs[@]}" -o "${TMP_ZIP}" -A "itzg/minecraft-bedrock-server" -fsSL "${DOWNLOAD_URL}"; then
+  if ! curl "${curlArgs[@]}" -H "user-agent: itzg/docker-minecraft-bedrock-server" -o "${TMP_ZIP}" -A "itzg/minecraft-bedrock-server" -fsSL "${DOWNLOAD_URL}"; then
```

Comment:
The -A already sets user agent.

Reply:
Perfect, thanks! Will close this PR then.